### PR TITLE
Update plugin-deployit-plugin.yml

### PR DESCRIPTION
**Pull request by saraswat2385**

### CODE REVIEW

**Comment by orrc** [comment]
@bdevic or @ravan should confirm that these permission changes are desired.

`msaraswat` isn't a valid account on jenkins.io, so that username needs to be fixed or removed.

**Comment by orrc** [comment]
(There also isn't a [default assignee on JIRA][0] for this plugin; it would be nice if you could name a jenkins.io account that we can set as the default)

[0]:https://issues.jenkins-ci.org/projects/JENKINS?selectedItem=com.atlassian.jira.jira-projects-plugin:components-page&contains=deployit

File: permissions/plugin-deployit-plugin.yml [hunk 1]
```diff
@@ -4,4 +4,6 @@ paths:
 - "com/xebialabs/deployit/ci/deployit-plugin"
 developers:
 - "bdevic"
-- "ravan"
\ No newline at end of file
+- "ravan"
+- "msaraswat"
```
Comment (batmat) [inline permissions/plugin-deployit-plugin.yml]: non existing user

Comment (batmat) [inline permissions/plugin-deployit-plugin.yml]: Woops, just saw @orrc did the same comment :)

**Comment by saraswat2385** [comment]
@batmat @orrc msaraswat is valid user see https://wiki.jenkins-ci.org/display/~msaraswat

**Comment by saraswat2385** [comment]
@batmat @orrc see https://github.com/jenkins-infra/repository-permissions-updater/commit/7878f2206f9ecd68f6be0d4c7e23dc5138504ad4 for more info

**Comment by bdevic** [comment]
@saraswat2385 and @adityakalia are the new plugin maintainers, so it's a valid change if the accounts are ok.

**Comment by daniel-beck** [comment]
For reference, this plugin is at https://github.com/jenkinsci/xldeploy-plugin – naming seems kind of crazy in that project so was a bit difficult to find.

**Comment by daniel-beck** [comment]
All referenced accounts are present in Artifactory.

**Comment by batmat** [comment]
WTF then how come https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=msaraswat leads to an error?

Aren't the accounts centralized from LDAP? WDYT @rtyler ?

**Comment by daniel-beck** [comment]
@batmat Maybe same behavior as Artifactory, requires initial login to exist? And the user was never logged in to JIRA?

**Comment by batmat** [comment]
No idea, TBH. That would be @rtyler if someone has to know it I guess.

**Comment by batmat** [comment]
@saraswat2385 Quick question, how did you create your Jenkins account? Did you go through accounts.jenkins.io? 
And if so, do you remember when that was, and if you ever logged in JIRA with that account since?

Thank you for the clarification.

**Comment by saraswat2385** [comment]
@batmat i created my account using https://accounts.jenkins.io

**Pull request merged**